.github/workflows/pr-plugin-build.yml

```
@@ -0,0 +1,154 @@
+        run: |
+          # Generate version based on PR number and short SHA
+          VERSION="pr.${{ github.event.pull_request.number }}.$(git rev-parse --short HEAD)"
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+          echo "Generated version: $VERSION"
+      
+      - name: Create plugin package
+        if: steps.changed-files.outputs.has_changes == 'true'
+        run: |
+          # Create build directory
+          mkdir -p build/usr/local
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)